tests/test_code_generator_libcpp.py

```diff
@@ -176,9 +176,6 @@ def test_libcpp():
     assert list(out.items()) == [(1, 2.0)]
 
     out = t.process13(libcpp.EEE.A, 2)
-    print (list(out.items()))
-    print ([(libcpp.EEE.A, 2)])
-    print (list(out.items()) == [(libcpp.EEE.A, 2)])
     assert list(out.items()) == [(libcpp.EEE.A, 2)]
```

Comment:
can we test the type stubs / typelist generation somehow?

Reply:
have to think about it. probably. you could at least compare the resulting files